examples/hello-world/src/test-api.js

```diff
@@ -62,11 +62,10 @@ export default class TestApi extends FreshDataApi {
 		}
 	}
 
-	static selectors = {
-		getGreetings: ( getData, requireData ) => ( requirement ) => {
+	selectors = {
+		getGreetings: ( getResource, requireResource ) => ( requirement ) => {
 			const resourceName = 'greetings';
-			requireData( requirement, resourceName );
-			return getData( resourceName ) || [];
+			return requireResource( requirement, resourceName ).data || [];
```

Comment:
just a random nitpick but in cases like these where we return a default value it can be helpful to define it in the model scope like `const emptyArray = []` and return `emptyArray`

why? because `[]` is a new literal on every call so React treats it as shallow-different

Reply:
Okay, so you're saying to put `const emptyArray = []` as a global to the `test-api.js` file and reuse it there?

FWIW, I wouldn't recommend checking equality on an array anyway, I'd recommend using `0 === array.length` instead.

Reply:
I didn't notice this was a test file

the point is that if this data feeds into React components then unless we tell React to check for value equality it will assume the value is new and re-render even though nothing changed.